src/main/kotlin/net/casual/arcade/border/CenterBorderState.kt

```diff
@@ -0,0 +1,17 @@
+package net.casual.arcade.border
+
+import net.minecraft.world.level.border.BorderStatus
+import org.joml.Vector2d
+
+interface CenterBorderState {
+
+
+
+    fun getCenter(): Vector2d
+
+    fun update(): CenterBorderState
+
+    fun onCenterChange()
```

Comment:
Center state doesn't need to be notified on when the center changes (it dictates this anyways)